src/components/frame/v5/pages/DashboardPage/partials/Menu/types.ts

```diff
@@ -0,0 +1,37 @@
+import { MessageDescriptor } from 'react-intl';
+
+export const MENU_ITEM_TYPE = {
+  BUTTON: 'button',
+  BUTTON_WITH_TOOLTIP: 'button-with-tooltip',
+  LINK: 'link',
+  DIVIDER: 'divider',
+} as const;
+
+export type MenuItemType = (typeof MENU_ITEM_TYPE)[keyof typeof MENU_ITEM_TYPE];
+
+export interface MenuItemList {
```

Comment:
MenuItemList -> MenuItem
This is the pattern that you should use. The component's name should be related to Colony Dashboard, because it's in its partials